_layouts/tutorial.html

```diff
@@ -48,10 +48,10 @@ <h1 class="post-title-main">{{ page.title }}</h1>
 
     <div class="tags">
         {% if page.tags != null %}
-        <b>Tags: </b>
         {% assign projectTags = site.data.tags.allowed-tags %}
         {% for tag in page.tags %}
         {% if projectTags contains tag %}
+        <b>Tags: </b>
```

Comment:
Won't this show the `Tags:` label multiple times? once per tag?

Reply:
This seems to be okay.
![screen shot 2017-11-06 at 5 09 25 pm](https://user-images.githubusercontent.com/8000644/32472027-44de1d6c-c315-11e7-952e-7ccf5d7e54c6.png)

Reply:
Weird! Are we sure that screenshot is using that _layout file. Can we do a quick sanity check please.

Reply:
Fixed!